radio/src/gui/gui_common_arm.cpp

```diff
@@ -278,16 +278,16 @@ bool isSwitchAvailable(int swtch, SwitchContext context)
   bool negative = false;
 
   if (swtch < 0) {
-    negative = true;
     if (swtch == -SWSRC_ON || swtch == -SWSRC_ONE) {
       return false;
     }
+    negative = true;
     swtch = -swtch;
   }
 
 #if defined(PCBSKY9X)
   if (swtch >= SWSRC_FIRST_SWITCH && swtch <= SWSRC_LAST_SWITCH) {
-    negative = negative;
+    (void)negative;
```

Comment:
`UNUSED()` is only defined in the third-party Lua source code.  Didn't seem relevant to include it here.  But it does the same thing.
`#define UNUSED(x)	((void)(x))`